### DISCUSSION TITLE
Unable to figure out where the orange background is being set?

### DISCUSSION THREAD

**Discussion by slice-shikharr** [JanDeDobbeleer/oh-my-posh#2631] [Troubleshoot] [answered]
Hi,

I am using https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/stelbent-compact.minimal.omp.json for my iTerm2 . I tried customizing some of the colours, but I can't figure out where this orange colour is being set whenever the state of the working Git directory is dirty.

in the below screenshot, the colours are fine when I am on master with no changes. But the moment I make any change, the Git segment turns orange.

Can someone please help me understand how to change this colour?
![image](https://user-images.githubusercontent.com/104886321/183416677-0424ff4c-ea21-4c8b-8dc2-f9e8a0ad13d5.png)

**Comment by JanDeDobbeleer** [comment, marked as answer]
You should have a look at `background_templates`. See [here](https://ohmyposh.dev/docs/configuration/colors#color-templates) for more information.

**Reply by slice-shikharr** [reply]
Hi @JanDeDobbeleer . Thank you for explaining that. I was able to successfully resolve the issue. I initially went down the wrong direction and thought it had something to do [with Powerline](https://ohmyposh.dev/docs/configuration/segment#powerline). So I was trying to figure out how[ Powerline was setup](https://powerline.readthedocs.io/en/master/configuration.html) and configured on my machine. I missed the concept of `background_templates` .

Marking issue as resolved. Able to modify the prompt for desired behaviour now.
